images/cli-artifacts/Dockerfile.rhel

```diff
@@ -4,11 +4,15 @@ FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
 WORKDIR /go/src/github.com/openshift/oc
 COPY . .
 RUN yum install -y --setopt=skip_missing_names_on_install=False gpgme-devel libassuan-devel
-RUN make cross-build-darwin-amd64 cross-build-windows-amd64 --warn-undefined-variables
+RUN make cross-build --warn-undefined-variables
 
 FROM registry.svc.ci.openshift.org/ocp/4.2:cli
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/darwin_amd64/oc /usr/share/openshift/mac/oc
 COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/windows_amd64/oc.exe /usr/share/openshift/windows/oc.exe
+COPY --from=builder /go/src/github.com/openshift/oc/_output/bin/linux_amd64/oc /usr/share/openshift/linux-x86_64/oc
```

Comment:
why rename here?  Can't we just stick with the GOARCH?

Reply:
I wasn't sure how to layout these directories so I went by https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.2/ (where uname-style arches are used instead of GOARCH-style names).  As they are new, they can be named as desired, I just need to keep the console-operator patch in sync.

Reply:
I don't see us porting `oc` from Go to another language in the near future, so I'd rather just:

```Dockerfile
COPY /go/src/github.com/openshift/oc/_output/bin/ /usr/share/openshift/
```

Folks who want a different naming scheme can translate; we don't have to translate for them here.